jsonnetfile.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
> Side note: There's no real reason to add jsonnetfile.json to .gitignore, as there's cases where you want to commit it (additional 3rd party Jsonnet dependencies).
>
> You can ignore it locally either in a global .gitignore (git config --global core.excludesfile = /path/to/global/gitignore and then add jsonnetfile.json there) or by adding it to the repo's .git/info/exclude.

My point wasn't that you should commit the auto-generated `jsonnetfile.json`, but that there may be cases where you want to commit a `jsonnetfile.json` containing custom dependencies in the future, and therefore the `jsonnetfile.json` shouldn't be added to the repo's `.gitignore`. If having the lile listed as "Untracked" in `git status` is an issue for you, there's multiple ways to ignore it locally (global gitignore, and `.git/info/exclude` in the repo).

Reply:
Ah, I misunderstood this